portlet-tck_3.0/driver/pom.xml

```diff
@@ -662,6 +662,14 @@
                            <value>${test.browser.webDriver}</value>

```

Comment:
@vsingleton "test.browser.webDriver" is the property that the Portlet 3.0 TCK already has for specifying a binary. For example, this is hard-coded:
`<test.browser.webDriver>c:/ntutil/chromedriver.exe</test.browser.webDriver>`

Is it necessary to introduce a new property named "test.browser.binary" in order to make this work? Or should we just rely on the developer overriding the value of "test.browser.webDriver" via the command-line?